sugarscape.py

```diff
@@ -985,6 +996,15 @@ def verifyConfiguration(configuration):
             print(f"Cannot provide {configuration['environmentMaxTribes']} tribes. Allocating maximum of {maxTribes}.")
         configuration["environmentMaxTribes"] = maxTribes
 
+    maxEthicalDecisionModels = 25
+    uniqueAgentDecisionModels = set(configuration["agentDecisionModels"])
+    numUniqueAgentDecisionModels = len(uniqueAgentDecisionModels)
+    if numUniqueAgentDecisionModels > maxEthicalDecisionModels:
+        if "all" in configuration["debugMode"] or "agent" in configuration["debugMode"]:
+            print(f"Cannot provide {len(configuration['agentDecisionModels'])} ethical models. Allocating maximum of {maxEthicalDecisions}.")
+        removeDecisionModels = uniqueAgentDecisionModels[maxEthicalDecisionModels:]
```

Comment:
Rather than removing decision models because we don't have enough colors, we should probably add more colors. This is a discussion to have with @nkremerh.

Reply:
We have 25 colors currently hardcoded into the GUI. We could add a mechanism to programmatically create colors, but I would worry about information overload for the user if we have an unlimited number of colors they could request.

I would recommend collapsing `maxTribes` and `maxEthicalDecisionModels` into one variable: `maxColors`.

Reply:
Combined.